python/lsst/ts/electrometer/controller.py

```diff
@@ -157,18 +158,18 @@ async def set_digital_filter(self, activate_filter, activate_avg_filter, activat
         activate_med_filter : bool
             Whether the median filter should be activated.
         """
-        filter = activate_filter
+        filter_active = activate_filter
```

Comment:
Why are you using the singleton comparison in the following code? And why are you using two if statements where you could simply do

```
filter_active = activate_avg_filter and activate_filter
```

?